lambdas/es/indexer/index.py

```diff
@@ -487,13 +487,19 @@ def retry_s3(
     else:
         arguments['IfMatch'] = etag
 
-    time_remaining = get_time_remaining(context)
```

Comment:
should also remove `context` argument to this function

Reply:
Done.